### PR TITLE
[12.0] [MIG] maintenance_equipment _status

### PR DESCRIPTION
**Pull request by NuriaXifre**
MIgration of the module maintenance_equipment_status to v12
cc ~ @ForgeFlow

### CODE REVIEW

**Review by etobella — APPROVED** [review]
No major changes. LGTM :+1:

**Comment by etobella** [comment]
Well, it is crazy to work with scrap, because two headers will be generated then..., maybe this header needs to be added on base_maintenance?
@lreficent

**Review by JordiBForgeFlow — APPROVED** [review]

File: maintenance_equipment_status/views/maintenance_equipment_views.xml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<odoo>
+
+    <!-- equipment : views -->
+    <record id="hr_equipment_view_form" model="ir.ui.view">
+        <field name="name">equipment.form</field>
+        <field name="model">maintenance.equipment</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form"/>
+        <field name="arch" type="xml">
+            <sheet position="before">
+                <header>
```
Comment (LoisRForgeFlow) [inline maintenance_equipment_status/views/maintenance_equipment_views.xml]: @etobella You mean this header?

Comment (etobella) [reply maintenance_equipment_status/views/maintenance_equipment_views.xml]: Yes

Comment (LoisRForgeFlow) [reply maintenance_equipment_status/views/maintenance_equipment_views.xml]: Ok, I agree.

@NuriaMartinXifre can you migrate base_maintenance and include the empty header for this view in that module? then, we'll need to include it here as a dependency and extend the view accordingly.

**Comment by etobella** [comment]
@NuriaMForgeFlow Fix the commit history please, as you have some merges on it and a `syntax fix`

**Comment by etobella** [comment]
@NuriaMForgeFlow Something weird happened with the commit history. Can you review it please?

**Comment by etobella** [comment]
Something weird happened with the commit 288676d7ca6bca1ffb431e81c1a207008b230d4d

**Comment by etobella** [comment]
Just a question, should we need to merge all the translation commits?
@LoisRForgeFlow

**Comment by LoisRForgeFlow** [comment]
@etobella we can squash them together if you want

**Comment by etobella** [comment]
Let's keep it this way
/ocabot merge

**Pull request merged**